cEP-0021.md

```diff
@@ -0,0 +1,266 @@
+# Debug and Profile Bears
```

Comment:
I don't want to hinder the linters I want to run the profiler on every method of linters which is getting called (except some wrapper like deprecate_settings or map_setting_to_aspect ) because linter bears are those bears which need more care than local bears , so don't you think If I will start insulating the functions of linters then It will not work the way we wanted.

Reply:
I have added the code to prevent it from moving into decorator like `map_setting_to_aspect` 

https://github.com/RaiVaibhav/coala/pull/1/files#diff-38a8902d01c1c77aeb2b6c313e737f2bR9

Reply:
I will update the cEP regarding this.